code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1010,22 +1018,14 @@
 							to_chat(cleaned_human, SPAN_WARNING("\The [src] runs its bottom mounted bristles all over you!"))
 		return
 
-/mob/living/silicon/robot/proc/self_destruct(var/anti_theft = FALSE)
-	if(anti_theft)
-		say("WARNING! Removal from NanoTrasen property detected. Anti-Theft mode activated. Unit [src] will self-destruct in five seconds.")
-		to_chat(src, SPAN_WARNING("All databases containing information related to NanoTrasen have been wiped!"))
-	else
-		say("WARNING! Self-destruct initiated. Unit [src] will self destruct in five seconds.")
```

Comment:
It seems you have not readded the `say("WARNING! Self-destruct initiated. Unit [src] will self destruct in five seconds.")` somewhere.
-> self_destruct() is also called if a borg is self_destructed from the robotics console, so that warning is quite important.

Reply:
A separate message is given when the detonate is activated, but I've modified both and added another one on the self_destruct proc to make it a bit clearer.